Documentation/configuration.md

```diff
@@ -130,6 +130,10 @@ To start etcd automatically using custom settings at startup in Linux, using a [
 + default: none
 + env variable: ETCD_DISCOVERY_PROXY
 
+##### -strict-reconfig-check
++ Reject reconfiguration requests if the requests can cause deadlock potentially if unstarted nodes have wrong peer URLs
```

Comment:
I think we can remove the peerURL part. We can catch the case that the reconfiguration might cause a potential quorum loss.